src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/Row.js

```diff
@@ -4,6 +4,10 @@ import tableStyles from './table.scss';
 import type {RowProps} from './types';
 
 export default class Row extends React.PureComponent<RowProps> {
+    static defaultProps = {
+        selected: false,
```

Comment:
Shouldn't the row also render the checkbox itself, if you are passing the properties this way?